src/routes/+page.svelte

```diff
@@ -7,22 +7,41 @@
 
 <section class="prose max-w-2xl p-8 dark:prose-invert sm:p-12 md:p-16">
     <h1 class="font-heading">Learn. Create. <em>Innovate.</em></h1>
+    <h2 class="font-heading">University of the Philippines Center for Student Innovations</h2>
     <p>
-        UP CSI is a service-oriented student organization aimed towards the enhancement of student
-        learning by working with real-world software projects.
+        A service-oriented organization aimed towards the enhancement of student learning from the
+        University of the Philippines Diliman, Department of Computer Science by working with
+        real-world projects.
     </p>
 </section>
 <section class="prose max-w-2xl max-w-full p-8 dark:prose-invert sm:p-12 md:p-16">
-    <h1 class="font-heading">What We Do</h1>
-    <p>
-        Our mission is to provide opportunities for students to enhance their software engineering
-        and their innovative thinking by:
-    </p>
+    <h1 class="font-heading">Our Mission</h1>
+    <ul>
+        <li>
+            Create development camps wherein students can enrich their skills in Software
+            Engineering.
+        </li>
+        <li>Have a diverse set of software projects to enhance the versatility of UP CSI.</li>
+        <li>
+            Showcase the quality projects produced by the Department of Computer Science (DCS) and
+            UP CSI and make them accessible to the public.
+        </li>
+    </ul>
+</section>
+<section class="prose max-w-2xl max-w-full p-8 dark:prose-invert sm:p-12 md:p-16">
+    <h1 class="font-heading">Our Vision</h1>
     <ul>
-        <li>Enhancing and upskilling student abilities through our developer training program.</li>
-        <li>Exploring avenues for industry partnerships.</li>
-        <li>Fulfilling the needs of the community through technological innovation.</li>
-        <li>Providing solutions to <em>your</em> problems.</li>
```

Comment:
I would like to keep these lines in some way, although not necessarily under the "Our Vision" section. These lines with the keywords "partnerships", "fulfilling needs", and usage of the second-person "you" , then directly followed by the contact form when scrolling downwards presents more appeal to prospective clients, in my opinion. :D

Reply:
Cc: @calebbun

What do you think?

Reply:
As a resolution, I have reinstated the **What We Do** section below **Our Vision**.